sarc/ldap/acquire.py

```diff
@@ -42,12 +42,16 @@ def run(
         cache_policy=cache_policy,
     )
 
-    # MyMila scraping is temporary disabled until we have a proper solution. 
-    # LD_users = fetch_mymila(
-    #     cfg,
-    #     LD_users,
-    #     cache_policy=cache_policy,
-    # )
+    # MyMila scraping "NotImplementedError" is temporary ignored until we have a working fetching implementation,
+    # or a working workaround using CSV cache.
+    with using_trace(
+        "sarc.ldap.acquire", "fetch_mymila", exception_types=(NotImplementedError,)
```

Comment:
Je l'ai fait comme ça, de manière à gérer les deux cas:
- en l'absence de fichier CSV de mymila (dans ce cas la lecture du cache échoue), fetch_mymila est appelée, une exception "Not Implemented" est lancée et gracieusement catchée, et MonMila est donc simplement ignoré
- si on a le CSV, il est manuellement placé dans le répertoire de cache et directement lu. 

Pour l'heure le traitement de ce fichier est l'objet d'une autre PR mais pas encore Ok. Donc on ne place simplement pas le CSV en prod et le code fonctionne en l'état.